querydsl-core/src/main/java/com/querydsl/core/support/CollectionAnyVisitor.java

```diff
@@ -110,7 +110,7 @@ protected Predicate exists(Context c, Predicate condition) {
             Path<?> parent = (Path<?>) expr.getMetadata().getParent().accept(this, context);
             expr = new PathImpl<Object>(expr.getType(), PathMetadataFactory.forCollectionAny(parent));
             EntityPath<?> replacement = new EntityPathBase<Object>(expr.getType(),
-                    ExpressionUtils.createRootVariable(expr));
+                    ExpressionUtils.createRootVariable(expr, replacedCounter++));
```

Comment:
Yes I mean the post increment operator.
I am also used to it, and figured that you were as well, but since it's a part of coding style, it's good to think it through.
I don't mind if we keep it, I just wanted to bring it up for discussion.
@johnktims you can join us as well.